src/main/java/io/strimzi/kafka/metrics/KafkaMetricsCollector.java

```diff
@@ -125,8 +120,17 @@ static MetricFamilySamples convert(String name, String help, KafkaMetric metric,
                             throw new IllegalStateException("Unexpected duplicate key " + v1);
```

Comment:
Sanitizing might result in collision. Hopefully it won't happen. I wonder if a WARN would be better that an exception.  I think as a user I would rather have metrics with an incomplete set of labels, rather than none. WDTY @mimaison

Reply:
Todo

Reply:
Yes agreed.

Reply:
Refactored to use WARN instead of an exception